lib/src/util/react_wrappers.dart

```diff
@@ -162,6 +162,15 @@ Map getProps(/* ReactElement|ReactComponent */ instance, {bool traverseWrappers:
 /// (and for the function itself, which is declared using `var` in react-dart).
 Element findDomNode(dynamic instance) => react_dom.findDOMNode(instance);
 
+/// Returns a a portal that renders [children] into a [container].
+///
+/// Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+///
+/// [children] can be any renderable React child, such as a [ReactElement], [String], or fragment.
+///
+/// See: <https://reactjs.org/docs/portals.html>
+ReactPortal createPortal(dynamic children, Element container) => ReactDom.createPortal(children, container);
```

Comment:
Just so I know - is this showing as an addition because it accidentally snuck into `3.0.0-wip` and we had to remove it?

Reply:
No, this was never exported from over_react so I'm adding it for the first time in PR.